lib/account/pages/account_page.dart

```diff
@@ -22,28 +23,92 @@ class _AccountPageState extends State<AccountPage> {
     AuthState authState = context.read<AuthBloc>().state;
     AccountState accountState = context.read<AccountBloc>().state;
 
-    return BlocConsumer<AuthBloc, AuthState>(
-      listener: (context, state) {},
-      builder: (context, state) {
-        if (authState.isLoggedIn && accountState.status == AccountStatus.success) return UserPage(userId: accountState.personView!.person.id, isAccountUser: true);
```

Comment:
There might be conflicting files here - I recently changed the account page to redirect to the UserPage since they share the same base. I think you might have pulled the `develop` branch when it was still in that phase of being transitioned!